### DISCUSSION TITLE
Getting `pwd` into statusline

### DISCUSSION THREAD

**Discussion by smacz42** [nvim-lualine/lualine.nvim#359] [Q&A] [answered]
I know that I can get the current directory with `:pwd` on the vim commandline, and that there is a function `getcwd()` that vim has. However, I don't know how - or even if I should - get that into the statusline. If not, I'd want to figure out where would be a good place to have that as an indicator.

**Comment by shadmansaleh** [comment, marked as answer]
TBH never felt like I needed a cwd indicator . I use [project.nvim](https://github.com/ahmedkhalf/project.nvim) that puts my cwd at project root based on which file I'm in.

If you want to place it in statusline or tabline with lualine you can use a simple function component like this
```lua
lualine_c = {function() return vim.fn.getcwd() end}
```

**Reply by smacz42** [reply]
Thanks! I do as well (vim-rooter, but will look into project.nvim) but always felt that I switched between projects enough that I needed 
some indicator. I am playing around with this as well as setting the title. It's harder when using Tmux, but I may end up putting it there.
